Clue/Classes/Modules/DeviceInfoModule.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DeviceInfoModule.swift
+//  Clue
+//
+//  Created by Andrea Prearo on 5/8/17.
+//  Copyright © 2017 Ahmed Sulaiman. All rights reserved.
+//
+
+import Foundation
+
+/// `DeviceInfoModule` is a info module (with static, one-time informations) for current devices' information recording on start.
+public class DeviceInfoModule: NSObject, CLUInfoModule {
+    fileprivate let writer: JSONWriter
+
+    // MARK: - Lifecycle
+    public required init(writer: CLUWritable) {
+        self.writer = writer as! JSONWriter
+    }
+
+    // MARK: - Public Methods
+    public func recordInfoData() {
+        let deviceProperties = getDeviceProperties()
+        writer.startWriting()
+        writer.append(json: deviceProperties)
+        writer.finishWriting()
+    }
+}
+
+// MARK: - Private Methods
+fileprivate extension DeviceInfoModule {
+    func getDeviceProperties() -> [AnyHashable: Any] {
```

Comment:
What do you think about encapsulation of this function into the separate `UIDevice` extension? 
Something like we did with `UIView` category to parse its properties for example.
```swift
let device = UIDevice.current
var deviceProperties = device.clueProperties() // which will return dictionary with properties
```